test/Microsoft.DotNet.ApiCompatibility.Tests/Microsoft.DotNet.ApiCompatibility.Tests.csproj

```diff
@@ -17,6 +17,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(RepoRoot)src\Compatibility\ApiCompat\Microsoft.DotNet.ApiCompatibility\Microsoft.DotNet.ApiCompatibility.csproj" />
+    <ProjectReference Include="$(RepoRoot)src\Compatibility\Microsoft.DotNet.ApiSymbolExtensions\Microsoft.DotNet.ApiSymbolExtensions.csproj" />
```

Comment:
I'm a bit surprised to see this here. Was this P2P really necessary? ApiSymbolExtension should already be a transitive dependency of the above ApiCompatibility.csproj P2P.

Reply:
I was seeing a failure. But I removed it and it's gone. Unsure why that failure was showing up in the first place.